tests/test_hv.py

```diff
@@ -4,28 +4,34 @@
 from unittest.mock import patch
 
 
-def test_arch():
-    hv = virt_lightning.LibvirtHypervisor("test:///default")
+def test_arch(hv):
     assert hv.arch == 'i686'
```

Comment:
well, it might not work for non linux systems.

Reply:
It should, see: https://libvirt.org/drvtest.html